airbyte-commons-protocol/src/main/java/io/airbyte/commons/protocol/migrations/ConfiguredAirbyteCatalogMigration.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.commons.protocol.migrations;
+
+public interface ConfiguredAirbyteCatalogMigration<PreviousVersion, CurrentVersion> extends Migration {
+
+  /**
+   * Downgrades a ConfiguredAirbyteCatalog to from the new version to the old version
```

Comment:
typo in this javadoc, the first `to` doesn't apply

Reply:
fixed